sputnik-staking/src/lib.rs

```diff
@@ -162,7 +162,7 @@ mod tests {
     use near_sdk::json_types::U64;
     use near_sdk::test_utils::{accounts, VMContextBuilder};
     use near_sdk::testing_env;
-    use near_workspaces::types::NearToken;
+    use near_token::NearToken;
```

Comment:
NearToken is re-exported by near-sdk, so you don't need to explicitly depend on near-token crate:

```suggestion
    use near_sdk::NearToken;
```